modelon/impact/client/entities/workspace.py

```diff
@@ -489,19 +449,13 @@ def create_experiment(
         """Creates an experiment. Returns an Experiment class object.
 
         Args:
-
-            definition:
-                An parametrized experiment definition class of type
-                modelon.impact.client.experiment_definition.base.SimpleModelicaExperimentDefinition
-                or
-                modelon.impact.client.experiment_definition.base.SimpleFMUExperimentDefinition.
+            definition: An parametrized experiment definition class of type
+                SimpleModelicaExperimentDefinition or SimpleFMUExperimentDefinition.
             user_data:
```

Comment:
need reformatting?

Reply:
Hmm This is inline with the [spec](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)? The second line should be indented one level inside if the parameter description spans over multiple lines.